src/benchq/resource_estimation/graph_compilation.py

```diff
@@ -127,21 +127,18 @@ def get_resource_estimations_for_graph(
     n_measurements_steps = len(scheduler_only_compiler.measurement_steps)
     total_time = calculate_wall_time(
         min_viable_distance,
-        n_nodes,
+        n_measurements_steps,
```

Comment:
This actually breaks the case where we estimate from subgraphs. We can't make this change here. If you want to chat about it we can have a call.